modules/images-other-jenkins-oauth-auth.adoc

```diff
@@ -18,12 +18,28 @@ Jenkins supports both browser and non-browser access.
 
 Valid users are automatically added to the Jenkins authorization matrix at log
 in, where {product-title} `Roles` dictate the specific Jenkins permissions the
-users have.
+users have.  The `Roles` used by default are the `admin`, `edit`, and `view` 
+`Roles` in the `Project` or namespace that Jenkins is running in.
 
 Users with the `admin` role have the traditional Jenkins administrative
 user permissions. Users with the `edit` or `view` role have progressively
 fewer permissions.
 
+Those defaults noted, the which set of {product-title} `Roles` and which Jenkins permissions
```

Comment:
Suggested change: 

"The default {product-title} `admin`, `edit`, and `view` `Roles` and the Jenkins permissions
those `Roles` are assigned in the Jenkins instance are configurable."

Reply:
part of next push